File: intFahrToCelcv1.c
#include <stdio.h>

//This programm prints the integer values of the Fahrenheit-Celcius table for
//the range 0 bis 300 in Fahrenheit.

int main(int argc, char const *argv[])
{
  int fahr, celcius;
  int lowerLimit, upperLimmit, stepSize;

  lowerLimit = 0;     //Lower limit of temperature table
  upperLimmit = 300;  //Upper limit of the table
  stepSize = 20;      //step size, wie genannt

  fahr = lowerLimit;
  while (fahr <= upperLimmit)
  {
    celcius = 5 * (fahr-32) / 9;
    printf("%d\t%d\n", fahr, celcius);
    fahr = fahr + stepSize;
  }

  return 0;
}

File: intFahrToCelcv1.rs
#![allow(non_snake_case)]

fn main()
{
    let lowerLimit:i64 = 0;     //Lower limit of temperature table
    let upperLimmit:i64 = 300;  //Upper limit of the table
    let stepSize:i64 = 20;      //step size, wie genannt

    let mut fahr:i64 = lowerLimit;
    while fahr <= upperLimmit
    {
        let celcius = 5 * (fahr-32) / 9;
        //println!("{}    {}", fahr, celcius);I didn't realise that it worked this way! I didn't really specify what goes where!
        println!("{fahr:width$}{celcius:width$}", width = 7);
        fahr = fahr + stepSize;
    }
}
